jmxtrans-output/jmxtrans-output-gelf/src/test/java/com/googlecode/jmxtrans/model/output/gelf/GelfWriterFactoryTests.java

```diff
@@ -227,7 +227,7 @@ public void testNonDefaultsWriterFactory() throws ValidationException {
 		assertThat(
 			gelfWriterFactory.getGelfConfiguration().getTlsTrustCertChainFile()
 				.getPath()
-		).isEqualTo("/somefile")
+		).isEqualTo(File.separator + "somefile")
```

Comment:
I do find the second version (`hasName()`) slightly nicer. But honestly, either is fine by me.